### PR TITLE
added DAO implementation

### PR DESCRIPTION
**Pull request by oroboro-cpu**

### CODE REVIEW

**Review by yurwing — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    public void handle() {
+        BetDao betDao = new BetDaoImpl();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's have this dependency as a field

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            try {
+                String[] data = command.split(" ");
+                int value = Integer.parseInt(data[0]);
+                double risk = Double.parseDouble(data[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
+                System.out.println("Please write correct input data");
+            } catch (ArrayIndexOutOfBoundsException e) {
+                System.out.println("Please write correct input value and risk");
+            }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: can we use only one catch block? the messages are pretty the same

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public class UserDaoImpl implements UserDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/UserDaoImpl.java]: redundant empty line

File: src/main/java/core/basesyntax/model/Bet.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        return risk;
+    }
+
+    public void setRisk(int risk) {
+        this.risk = risk;
+    }
+
+    @Override
+    public String toString() {
+        return "model.Bet{"
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/Bet.java]:
```suggestion
        return "Bet{"
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**